js/tweek-client/src/utils.ts

```diff
@@ -2,6 +2,16 @@ export function captialize(string) {
   return string.charAt(0).toUpperCase() + string.slice(1);
 }
 
+export const requestTimeout = (timeoutInMillis): Promise<Response> => {
```

Comment:
maybe make this `(timeout, fetch) => (input, init) => Promise.race(...)` to create a fetch with timeout